webpack/constants.js

```diff
@@ -21,7 +21,8 @@ const BABEL_INCLUDE_NODE_MODULES = [
     'get-random-values',
     'sieve.js',
     'pm-srp',
-    'react-components'
+    'react-components',
+    'key-transparency-web-client'
```

Comment:
it's out of the scope here, but why don't we prefix it via proton-

Reply:
this is something I would like to do generally when/if we do the monorepo

so everything would be prefixed with 

@proton/ui
@proton/kt
@proton/pack
...